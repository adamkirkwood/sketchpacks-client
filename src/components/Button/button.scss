.button {
  width: 16.0rem;
}
