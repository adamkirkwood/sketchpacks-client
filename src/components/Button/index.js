import React from 'react'

const Button = ({ actionVerb, onClick, className }) => (
  <button className={className} onClick={onClick}>{actionVerb}</button>
)

export default Button
