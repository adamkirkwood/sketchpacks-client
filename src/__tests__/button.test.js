import React from 'react'
import Button from '../components/Button'

describe('Button', () => {
  it('should render a button element', () => {
    const wrapper = shallow(
        <Button actionVerb={'Install'} />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
