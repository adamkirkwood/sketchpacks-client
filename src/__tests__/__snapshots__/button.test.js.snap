// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render a button element 1`] = `
<button>
  Install
</button>
`;
